### PR TITLE
Check people filter

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: features/environment.py [hunk 1]
```diff
@@ -42,12 +43,13 @@ def before_all(context):
...
     context.driver = webdriver.Chrome()
     context.wait = ui.WebDriverWait(context.driver, 10)
-
     context.login_page = LoginPage(context)
     context.authorized_page = AuthorizedPage(context)
     context.lobby_page = LobbyPage(context)
     context.api_page = ApiPage(context)
     context.settings_page = SettingsPage(context)
+    context.people_page = PeoplePage(context)
+    context.wait = ui.WebDriverWait(context.driver, 10)
```
Comment (helltepakcer) [inline features/environment.py]: I think at line 45 we got somethink similar:)

File: pages/authorized_page.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # -*- coding: UTF-8 -*-
 
 from .base_page import Page
-
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
```
Comment (helltepakcer) [inline pages/authorized_page.py:6]: PEP8 require

File: pages/people_page.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            print('\tAdmins found correctly')
+            pass
+        else:
+            print("Admins not found")
+
+    def show_all_users(self):
+        self.context.driver.find_element_by_id('show_all_users').click()
+        self.context.wait.until(EC.visibility_of_element_located((By.XPATH, '//ol[@class="aui-nav-pagination"]')))
+
+    def we_see(self):
```
Comment (helltepakcer) [inline pages/people_page.py]: From method name I don't understand what it gonna doing, please provide more correct name.

File: pages/people_page.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                self.context.wait.until(EC.element_to_be_clickable((By.XPATH, '//span[@title="Clear search"]')))
+                self.context.driver.find_element_by_xpath('//span[@title="Clear search"]').click()
+                pass
+        except:
+            print("\tData filtered correctly. Incorrect result was not displayed")
+            self.context.wait.until(EC.element_to_be_clickable((By.XPATH, '//span[@title="Clear search"]')))
+            self.context.driver.find_element_by_xpath('//span[@title="Clear search"]').click()
+
+
+
```
Comment (helltepakcer) [inline pages/people_page.py]: PEP8

**Review by helltepakcer — CHANGES_REQUESTED** [review]

File: pages/authorized_page.py (hunk 2 above)
Comment (ghost) [reply pages/authorized_page.py:6]: Done

File: features/environment.py (hunk 1 above)
Comment (ghost) [reply features/environment.py]: Done

**Review by helltepakcer — APPROVED** [review]

**Pull request merged**